MIGRATION.md

```diff
@@ -0,0 +1,16 @@
+#Migrate AWS RDS databases to use different private subnets
+
+This procedure explains how to migrate AWS RDS databases, deployed with ref architecture, from using separate, private DB subnets to (re-)using 
+private subnets already deployed with VPC. This only applies to already existing deployments, new deployments are not affected.
+The change was introduced with v0.2.0 of reference architecture.
+
+##Steps:
+1. Check your existing deployment and make sure everything works etc. (no issues, pods not starting and so on)
+2. Delete keycloak and simphera databases, making sure that final snapshot is created
```

Comment:
Can you provide more details to create a snapshot.

Reply:
It's a default option (automatically selected) when deleting a DB, no special input is needed.

Reply:
You could add a hint like "When deleting the database via the AWS Console, the option to create a snapshot is offered."

Reply:
Done